libs/tools/barista/src/builder/strapi.ts

```diff
@@ -64,6 +65,10 @@ export const strapiBuilder: BaPageBuilder = async (
     return [];
   }
 
+  if (next) {
+    TRANSFORMERS.push(headlineClassTransformer);
+  }
```

Comment:
We don't necessarily need to do this only for next. We could do this for the current barista outlet as well.

Reply:
Agree with tom! :)

Reply:
Will there be changes in the styling of barista concerning headlines being styled with classes instead of the current usage of the element selectors?

Reply:
yes, there should be changes to the stylings of headlines in barista. 
We want to decouple the semantics from the styling eventually.

Reply:
@tomheller but if we want to decuple styling from semantics we should not name the classes `h1` I would go then with something different that is not that much tied to the semantics